simplegrid/wolf.py

```diff
@@ -31,6 +31,8 @@ def step(self, observation):
                 if 0 < dist <= view_distance:
                     if value == MapFeature.ROCK.index:
                         reward = -1
+                    elif value == MapFeature.WATER.index:
```

Comment:
Currently trying to go into the water doesn't kill the Wolf, so maybe the same value as for rock would be ok?